### ISSUE TITLE
interactive_plot breaks if f returns multiple lines

### ISSUE THREAD

**Issue by ianhi** [ianhi/mpl-interactions#30]
```python
beta = np.linspace(4,10,)
x = np.linspace(0,2*np.pi)
other_param = np.array([1,2])[None,:]
def f(x, beta):
    return np.sin(x*beta)[:,None]* other_param
print(f(x,beta[10]).shape)
interactive_plot(f,x=x,beta =beta)
```
![Peek 2020-07-31 12-18](https://user-images.githubusercontent.com/10111092/89055157-f53fb600-d327-11ea-83b4-1855718a9f1b.gif)

**Comment by ianhi** [comment]
at the very least it should fail more gracefully

**Comment by ianhi** [comment]
fixed/rendered obsolete by #119
